macGist/Keychain.swift

```diff
@@ -7,6 +7,59 @@ struct KeychainConfiguration {
     static let serviceName = "macGist"
     static let accessGroup = "com.idevzilla.macGist"
     static let account = "macGistAccount"
+    
+    struct GitHub {
+        static let serviceName = "GitHub"
+        static let account = "GitHubAccount"
+        
+        struct Gist {
+            
+            enum Account: String {
+                case clientId
+                case clientSecret
+                
+                func query() -> [String:AnyObject] {
+                    let query = KeychainConfiguration.keychainQuery(withService: KeychainConfiguration.GitHub.serviceName,
+                                                             account: self.rawValue,
+                                                             accessGroup: KeychainConfiguration.accessGroup)
+                    return query
+                }
+            }
+        }
+        
+        static func query() -> [String:AnyObject] {
+            let query = KeychainConfiguration.keychainQuery(withService: KeychainConfiguration.GitHub.serviceName,
+                                                     account: KeychainConfiguration.GitHub.account,
+                                                     accessGroup: KeychainConfiguration.accessGroup)
+            return query
+        }
+    }
+    
+    static func query() -> [String:AnyObject] {
```

Comment:
Why do we have two implementations of this method?

Reply:
Removed implementation from `Keychain`.